package.json

```diff
@@ -25,5 +25,8 @@
   "homepage": "https://github.com/Sebasgl23/cypress-training#readme",
   "devDependencies": {
     "cypress": "^10.3.0"
+  },
+  "dependencies": {
+    "typescript": "^4.7.4"
```

Comment:
Typescript should be a dev dependency

Reply:
to fix it, run: `npm install --save-dev typescript`

Reply:
Thanks Holmes and Nicolas